homeassistant/components/solarlog/sensor.py

```diff
@@ -96,6 +96,15 @@ def state(self):
         """Return the state of the sensor."""
         return self._state
 
+    @property
+    def device_info(self):
+        """Return the device information."""
+        return {
+            "identifiers": {(DOMAIN, self.device_name)},
```

Comment:
Identifiers should be unique.  You could use the unique id here instead.
```suggestion
            "identifiers": {(DOMAIN, f"{self.entry_id}_{self.sensor_key}")},
```